src/Components/Facility/PatientNotesSlideover.tsx

```diff
@@ -34,6 +35,9 @@ export default function PatientNotesSlideover(props: PatientNotesProps) {
   const [patientActive, setPatientActive] = useState(true);
   const [reload, setReload] = useState(false);
   const [focused, setFocused] = useState(false);
+  const [reply_to, setReplyTo] = useState<PaitentNotesReplyModel | undefined>(
+    undefined,
+  );
```

Comment:
same here...

also consider abstracting the repeated logic to a common component. (To be done in upcoming PRs for enhancing doctor notes)